MediaWikiTelegramBot.js

```diff
@@ -46,6 +60,7 @@ try {
     console.log( e );
 }
 
+// Create Telegram bot
```

Comment:
I'd say "Telegram bot object", or remove the comment entirely, because it's quite self-explanatory.